tests/field/test_mgrid.py

```diff
@@ -1,9 +1,8 @@
 #!/usr/bin/env python
 
 import unittest
-import os
```

Comment:
Did os fail in same particular case? Is it why it was removed?

Reply:
most tests use `monty.ScratchDir` but the `test_mgrid` used `tempfile` and `os.chdir`, without changing back. 

This led my subsequent tests to fail mysteriously as the directory that the test-kernel had moved into had been deleted. 

Reverted this test to the more readable and less error-prone `moty.ScratchDir`, and therefore `os` import was also no longer needed to effect the `chdir`, so I removed it.